tailwind.config.js

```diff
@@ -55,7 +56,7 @@ module.exports = {
       },
       animation: {
         borderpulse: "border-pulse 3s ease-in-out infinite",
-        "borderpulse-blue": "border-pulse-blue 3s ease-in-out infinite",
+        "bordezrpulse-blue": "border-pulse-blue 3s ease-in-out infinite",
```

Comment:
The "z" slipped in by mistake?

Reply:
It seems so! ':D I will fix it!

Reply:
I removed the borderpulse-blue and bg-pulse animation as they weren't used anymore.